distributions/openhab/src/main/resources/bin/update.lst

```diff
@@ -103,6 +103,9 @@ ALERT;JavaScript NashornJS: NashornJS has been removed from core and isn't inclu
 ALERT;JavaScript Scripting Automation: The old "metadata" and "itemchannellink" APIs have been replaced by a new API with extended functionality on the "items" namespace.
 ALERT;JavaScript Scripting Automation: ItemHistory: historicState, maximumBetween, maximumSince, minimumBetween, minimumSince & previousState now return a HistoricItem instead of just the state. previousStateTimestamp has been removed, the timestamp is available on the HistoricItem.
 
+[4.1.0]
+ALERT;Twitter Binding has become X. All related references to Twitter and Tweets have been replaced, please check your configuration (rules, actions, channels etc).
+
```

Comment:
The diff looks strange, I think you need to rebase? The current version looks like this:
https://github.com/openhab/openhab-distro/blob/0811429c44cc8ec8925265e2a423ae2767d9f0b9/distributions/openhab/src/main/resources/bin/update.lst#L95-L128